config/workers.coffee

```diff
@@ -8,6 +8,11 @@ module.exports = (KONFIG, options, credentials) ->
   nodeProgram = if options.watchNode then './watch-node' else 'node'
 
   workers =
+    nginx:
+      group: 'webserver'
+      supervisord:
+        command: 'nginx -c %(ENV_KONFIG_PROJECTROOT)s/nginx.conf'
```

Comment:
I don't recall. I suppose it was just to have a shorter path.

Stop signal is a good catch. Also, that reminded me an old observation. I think `supervisorctl shutdown` kills the subprocesses because it is much faster than `supervisorctl stop all`. I'll dig into it.